simulation/test_dht_storage.cpp

```diff
@@ -88,7 +88,7 @@ namespace
 	}
 }
 
-void timer_tick(std::unique_ptr<dht_storage_interface> s
+void timer_tick(std::unique_ptr<dht_storage_interface>& s
```

Comment:
technically this should be raw pointers.. again, no need to change if this passes CI